### PR TITLE
Shorten the relative time of envelopes

### PR DESCRIPTION
**Pull request by ChristophWurst**
Fixes https://github.com/nextcloud/mail/issues/6972

Bildschirmfoto vom 2022-08-12 13-28-46

### CODE REVIEW

File: src/util/shortRelativeDatetime.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import curry from 'lodash/fp/curry'
+import moment from '@nextcloud/moment'
+
+export const shortDatetime = curry((ref, date) => {
```
Comment (ChristophWurst) [inline src/util/shortRelativeDatetime.js:25]: curried to make testing life easier :sunglasses:

**Review by GretaD — APPROVED** [review]
smooth

**Comment by GretaD** [comment]
test has a problem, checking

**Comment by GretaD** [comment]
test fixed

**Pull request merged**